pkg/controllers/tagging/tagging_controller.go

```diff
@@ -64,6 +65,7 @@ const (
 	taggingControllerLabelKey = "k8s.io/cloud-provider-aws"
 
 	maxRequeuingCount = 9
+	workerQueueCount  = 10
```

Comment:
This seems like a useful config option, similar to `--concurrent-node-syncs`, `--concurrent-service-syncs`, etc.